policy/HscMapper.yaml

```diff
@@ -642,3 +642,7 @@ datasets:
   qaTableVisit:
     template: plots/%(filter)s/tract-%(tract)d/visit-%(visit)d/%(description)s.parq
 
+  jointcal_wcs:  # astrometric calibration produced by jointcal/meas_mosaic
+    template: jointcal-results/%(filter)s/%(tract)04d/jointcal_wcs-%(visit)07d-%(ccd)03d.fits
+  photoCalib:  # photometric calibration produced by jointcal/meas_mosaic
+    template: jointcal-results/%(filter)s/%(tract)04d/jointcal_photoCalib-%(visit)07d-%(ccd)03d.fits
```

Comment:
Why not put these up at line ~356 along with the deprecated **photoCalib**?  That way the reader sees the newly favored dataset right away.